.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Erlang CI
+on: push
+jobs:
+  Test:
+    strategy:
+      matrix:
+        erlang: ['24']
+    runs-on: ubuntu-latest
+    container:
+      image: erlang:${{ matrix.erlang }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        env:
+          cache-name: rebar3
+        with:
+          path: |
+            _build
+            ~/.cache/rebar3
+          key: ci-${{runner.os}}-${{env.cache-name}}-erlang_${{matrix.erlang}}-rebar3-${{hashFiles('rebar.lock')}}
+          restore-keys: |
+            ci-${{runner.os}}-${{env.cache-name}}-erlang_${{matrix.erlang}}-rebar3
+            ci-${{runner.os}}-${{env.cache-name}}-erlang_${{matrix.erlang}}
```

Comment:
From my testing, the restore-keys logic coupled with our setup makes it so upgrading the lockfile will make the CI keep using stale dependencies from the cache.
```suggestion
```

Reply:
Nice catch! Looking for a backup cache while the dependencies have changed would be counter-intuitive. Fixed.